packages/lesswrong/components/common/HomeLatestPosts.tsx

```diff
@@ -36,7 +36,7 @@ const styles = (theme: ThemeType): JssStyles => ({
   }
 })
 
-const latestPostsName = forumTypeSetting.get() === 'EAForum' ? 'Frontpage Posts' : 'Latest'
+const latestPostsName = forumTypeSetting.get() === 'EAForum' ? 'Posts' : 'Latest'
```

Comment:
Why change this wording? Since we refer to "frontpage" in a few other places (like when saving a post), I would suggest keeping it for consistency.

Reply:
I thought it was redundant because it's obvious that you're on the front page, but I don't have a super strong opinion, I'll change it back